README.md

```diff
@@ -77,14 +77,28 @@ Place it into project library path in your IDE.
 ## Usage
 
 1. [Class Tarantool](#class-tarantool)
-1. [Predefined Constants](#predefined-constants)
+2. [Predefined Constants](#predefined-constants)
+3. [Manipulation connection](#manipulation-connection)
+  * [connect](#connect)
+  * [disconnect, close](#disconnect-close)
+  * [authenticate](#authenticate)
+  * [flushSchema, flush_schema](#flushschema-flush_schema)
+  * [ping](#ping)
+4. [Database queries](#database-queries)
+  * [select](#select)
+  * [insert, replace](#insert-replace)
+  * [call](#call)
+  * [eval, evaluate](#eval-evaluate)
+  * [delete](#delete)
+  * [update](#update)
+  * [upsert](#upsert)
 
 ### Class Tarantool
 
 _**Description**_: Creates a Tarantool client
 
 ``` php
-tarantool_object = new Tarantool([host = 'localhost'[, port = 3301]])
+$tarantool_object = new Tarantool([$host = 'localhost'[, $port = 3301]])
```

Comment:
```php
$tarantool_object = new Tarantool([$host = 'localhost'[, $port = 3301]])
```
This line is not a correct syntax and it can't be copy-pasted. Moreover, few lines below there is the "*Example*" section with real-usage examples, so I don't see the point of having almost the same but broken syntax in the method description. I would rather delete it entirely or update it to the full method signature. I, personally, would choose the second option as I find these signatures very useful while reading [APIs](http://php.net/manual/en/mongodb-driver-query.construct.php) in php manual. But maybe it's just me ;)